sdk/contentsafety/ai-content-safety-rest/review/ai-content-safety.api.md

```diff
@@ -99,7 +99,7 @@ export interface AnalyzeImageDefaultResponse extends HttpResponse {
 // @public
 export interface AnalyzeImageOptions {
     categories?: string[];
-    image: ImageData;
+    image: ImageData_2;
```

Comment:
@joheredi @jeremymeng  do you have any idea why the api extractor result would change this way, I checked , there's really no duplicated models in it.

Reply:
could be api-extractor bug